src/Query/HostsFileExecutor.php

```diff
@@ -76,7 +76,7 @@ private function getIpFromHost($host)
             return inet_ntop(strrev($ip));
         } elseif (substr($host, -9) === '.ip6.arpa') {
             // IPv6: replace dots, reverse nibbles and interpret as hexadecimal string
-            $ip = @inet_ntop(hex2bin(strrev(str_replace('.', '', substr($host, 0, -9)))));
+            $ip = @inet_ntop(pack('H*', strrev(str_replace('.', '', substr($host, 0, -9)))));
```

Comment:
They both do exactly the same thing, except that `hex2bin()` received a number of changes over its lifetime and was introduced later than the `pack()` equivalent, hence why using `pack()` fixes an issue with the last test run :+1: